src/index.js

```diff
@@ -0,0 +1,58 @@
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+const scoreListDisplay = (scores) => scores.map((score) => `<li class="list-group-item d-flex justify-content-between align-items-start">${score.user}:${score.score}</li>`).join('');
+
+const createGame = async (gameName) => {
+  const response = await fetch(`${baseURL}games/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(gameName),
+  });
+
+  const gameId = await response.json();
+  return gameId;
+};
+
+const addScore = async (userName, gameId) => {
+  const response = await fetch(`${baseURL}games/${gameId}/scores/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(userName),
+  });
+
+  const result = await response.json();
+  return result;
+};
+
+const getScore = async (gameId) => {
+  const response = await fetch(`${baseURL}games/${gameId}/scores/`);
+  const result = await response.json();
+  return result;
+};
+
+const form = document.querySelector('#form');
+const scoreList = document.querySelector('#scoreList');
+const refreshButton = document.querySelector('#refreshBtn');
+const user = document.querySelector('#name');
+const score = document.querySelector('#score');
+
+window.addEventListener('DOMContentLoaded', async () => {
+  const game = { name: 'My cool new game' };
+  const { result } = await createGame(game);
+  const gameId = result.split(' ')[2];
+
+  form.addEventListener('submit', async (event) => {
+    event.preventDefault();
+    const userName = { user: user.value, score: score.value };
+    await addScore(userName, gameId);
+  });
+
+  refreshButton.addEventListener('click', async () => {
+    const result = await getScore(gameId);
+    scoreList.innerHTML = scoreListDisplay(result.result);
+  });
+});
```

Comment:
- [ ] HI @elmar8287, very nice work so far! Currently, the Leaderboard is emptied when the page is loaded or reloaded. It only populates the leaderboard with scores when the refresh button is clicked. Kindly make changes so that the LeaderBoard is populated by scores data from the API during page load and reload. :pray:

Reply:
Dear @henatan99,

Thank you for your review. I have fixed the comments you mentioned. Now, it is displaying data during page load and reload

My regards,

Elmar

Reply:
Very well done!!! :+1: :clap: :clap: